var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -15,60 +15,67 @@ class Esmf(MakefilePackage):
     software infrastructure for building and coupling weather, climate, and
     related Earth science applications. The ESMF defines an architecture for
     composing complex, coupled modeling systems and includes data structures
-    and utilities for developing individual models."""
+    and utilities for developing individual models.
+    The National Unified Operational Prediction Capability (NUOPC) Layer
+    defines a common model architecture to support interoperable ESMF components.
+    The NUOPC Layer is included with the ESMF package."""
 
-    homepage = "https://www.earthsystemcog.org/projects/esmf/"
+    homepage = "https://earthsystemmodeling.org/"
     url = "https://github.com/esmf-org/esmf/archive/ESMF_8_0_1.tar.gz"
     git = "https://github.com/esmf-org/esmf.git"
 
-    maintainers("climbfuji", "jedwards4b", "AlexanderRichert-NOAA")
+    maintainers("climbfuji", "jedwards4b", "AlexanderRichert-NOAA", "theurich", "uturuncoglu")
 
     # Develop is a special name for spack and is always considered the newest version
     version("develop", branch="develop")
     # generate chksum with spack checksum esmf@x.y.z
     version("8.4.1", sha256="1b54cee91aacaa9df400bd284614cbb0257e175f6f3ec9977a2d991ed8aa1af6")
     version("8.4.0", sha256="28531810bf1ae78646cda6494a53d455d194400f19dccd13d6361871de42ed0f")
     version("8.3.1", sha256="6c39261e55dcdf9781cdfa344417b9606f7f961889d5ec626150f992f04f146d")
-    version("8.3.0", sha256="0ff43ede83d1ac6beabd3d5e2a646f7574174b28a48d1b9f2c318a054ba268fd")
-    version("8.3.0b09", commit="5b7e546c4b")
+    version(
+        "8.3.0",
+        sha256="0ff43ede83d1ac6beabd3d5e2a646f7574174b28a48d1b9f2c318a054ba268fd",
+        deprecated=True,
+    )
+    version("8.3.0b09", commit="5b7e546c4b", deprecated=True)
     version("8.2.0", sha256="3693987aba2c8ae8af67a0e222bea4099a48afe09b8d3d334106f9d7fc311485")
     version("8.1.1", sha256="58c2e739356f21a1b32673aa17a713d3c4af9d45d572f4ba9168c357d586dc75")
+    version(
+        "8.1.0",
+        sha256="367dc89ab8e273f6fbe3d63a64a9a6735cbaa77d5f144ebae4af198fee5bb203",
+        deprecated=True,
+    )
     version("8.0.1", sha256="9172fb73f3fe95c8188d889ee72fdadb4f978b1d969e1d8e401e8d106def1d84")
-    version("8.0.0", sha256="051dca45f9803d7e415c0ea146df15ce487fb55f0fce18ca61d96d4dba0c8774")
-    version("7.1.0r", sha256="ae9a5edb8d40ae97a35cbd4bd00b77061f995c77c43d36334dbb95c18b00a889")
+    version(
+        "8.0.0",
+        sha256="051dca45f9803d7e415c0ea146df15ce487fb55f0fce18ca61d96d4dba0c8774",
+        deprecated=True,
+    )
+    version(
+        "7.1.0r",
+        sha256="ae9a5edb8d40ae97a35cbd4bd00b77061f995c77c43d36334dbb95c18b00a889",
```

Comment:
I am not able to confirm this sha256.  I get `Could not find any remote versions for ESMF` when I run `spack checksum esmf@7.1.0r`.  If I add `url="https://github.com/esmf-org/esmf/archive/refs/tags/ESMF_7_1_0r.tar.gz," to the `7.1.0r` `version` directive, the sha256 is `e08f21544083dcbe162b472852e321f8df14f4f711f35508403d32df438367a7`.

Reply:
Thanks for checking. I simplified the tarball URL construction, and all tarballs now from GitHub (not sure why this was done differently before). Some checksums needed updating... should all be consistent now.